### PR TITLE
added new quote component - example 15

### PR DESCRIPTION
**Pull request by jahanvir**
This is new quote component named backgroung-image,

It sets image provided in field as background, it also allows text color modification.
image

### CODE REVIEW

**Comment by ayushsoni1010** [comment]
Wow @jahanvir, this looks pretty and unique component that you've crafted 🚀

File: quote.ftd [hunk 1]
```diff
@@ -2450,6 +2491,201 @@ border-bottom-left-radius.px: 8
...
+
+
+-- component quote-7-desktop:
+optional caption title:
+body body:
+ftd.image-src logo: $assets.files.static.quote.svg
+
+-- ftd.column:
+padding-vertical.px: 32
+padding-horizontal.px: 32
```
Comment (Arpita-Jaiswal) [inline quote.ftd:2539]:
```suggestion
padding.px: 32
```

File: quote.ftd [hunk 2]
```diff
@@ -2450,6 +2491,201 @@ border-bottom-left-radius.px: 8
...
+width: fill-container
+max-width.fixed.px: 800
+align-self: center
+margin-top.px: 24
+margin-bottom.px: 24
```
Comment (Arpita-Jaiswal) [inline quote.ftd:2544]:
```suggestion
margin-vertical.px: 24
```

File: quote.ftd [hunk 3]
```diff
@@ -2450,6 +2491,201 @@ border-bottom-left-radius.px: 8
...
+spacing.fixed.px: 8
+
+-- ftd.text: ~
+role: $inherited.types.copy-regular
+
+-- ftd.text: $quote-7-desktop.title
+role: $inherited.types.copy-regular
+
+
+-- end: ftd.row
```
Comment (Arpita-Jaiswal) [inline quote.ftd:2590]:
```suggestion
-- ftd.row:
align-self: center
spacing.fixed.px: 8
role: $inherited.types.copy-regular

-- ftd.text: ~

-- ftd.text: $quote-7-desktop.title

-- end: ftd.row
```

File: quote.ftd [hunk 4]
```diff
@@ -2450,6 +2491,201 @@ border-bottom-left-radius.px: 8
...
+
+
+-- component quote-7-mobile:
+optional caption title:
+body body:
+ftd.image-src logo: $assets.files.static.quote.svg
+
+-- ftd.column:
+padding-vertical.px: 16
+padding-horizontal.px: 16
```
Comment (Arpita-Jaiswal) [inline quote.ftd:2622]:
```suggestion
padding.px: 16
```

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: quote.ftd (hunk 3 above)
Comment (jahanvir) [reply quote.ftd]: Hi @Arpita-Jaiswal , Thank you for the review!! I have made the changes.

**Pull request merged**